### DISCUSSION TITLE
Show the 2FA "seed" code in addition to the QR code

### DISCUSSION THREAD

**Discussion by maxomatic458** [pelican-dev/panel#309] [Ideas]
might be useful for password managers

**Comment by TekExplorer** [comment]
> might be useful for password managers

That would be generally useful, especially if you wanted to hard-copy-backup your codes.

Granted, 1password can outright scan a QR code, but not many browser based managers can.

**Comment by rmartinoscar** [comment]
Already printed in [admin area](https://github.com/pelican-dev/panel/blob/main/app/Filament/Resources/UserResource/Pages/EditProfile.php#L184) and [client area](https://github.com/pelican-dev/panel/blob/main/resources/scripts/components/dashboard/forms/SetupTOTPDialog.tsx#L77)
I don't see what else do you want.

**Reply by TekExplorer** [reply]
It hadnt been when this discussion was started

**Reply by rmartinoscar** [reply]
It was added on Jun 1 hence my comment above

**Reply by TekExplorer** [reply]
If you knew that, then your comment should have been to suggest that the discussion should be closed due to the feature being added.

Instead, it just reads as... Argumentative? Rude? Something unpleasant like that. It gives the impression that the feature has already existed and that the creator of the discussion was wasting everyone's time, when that's not true.

"I don't see what else do you want" on top of "Already printed on..."

I dunno. Maybe I'm wrong. Read it out loud and see how it sounds to you.

**Reply by rmartinoscar** [reply]
Im sorry if that sounded rude, wasn't my attention it was a real question i wasn't sure i got the request right like if there was something missing from the current implementation.
I apologize for my wrong wording.
